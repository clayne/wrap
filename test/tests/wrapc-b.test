wrapc | /dev/null | -b@ | wrapc-b.txt | 0
