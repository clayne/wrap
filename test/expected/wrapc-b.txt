/**
 * Reads the configuration file.
 *
 * @param conf_file The full-path of the configuration file to read.  If NULL,
 * then the user's home directory is checked for the presence of the default
 * configuration file.  If found, that file is read.
 * @return Returns the full-path of the configuration file that was read.
 */
char const* read_conf( char const *conf_file ) {
  //
